### PR TITLE
Fix request in response documentation

### PR DESCRIPTION
**Pull request by Shmaltalk**
Add fmt_options, filters, and filter_match_types to documentation.

### CODE REVIEW

**Comment by Shmaltalk** [comment]
[ch3733]

File: source/includes/_search.md [hunk 1]
```diff
@@ -105,13 +105,24 @@ Values with a default (ie. page, num_results_per_page, section) will always be r
...
 ```
 {
 	"request": {
-		"page": 1
-		"num_results_per_page": 75
-		"term": "food"
+		"page": 1,
+		"num_results_per_page": 75,
+		"term": "food",
+		"section": "standard-section",
```
Comment (adetch) [inline source/includes/_search.md]: > "section": "standard-section",

to

> "section": "Products"

Comment (adetch) [reply source/includes/_search.md]: and could you add:
>>>
"sort_by": "relevance",
"sort_order": "descending",

Comment (Shmaltalk) [reply source/includes/_search.md]: fixed in 95e6056

**Pull request merged**